distribution/adapters/fuse-adapter-zip/src/main/resources/licenses/keycloak/licenses.xml

```diff
@@ -103,11 +103,11 @@
     <dependency>
       <groupId>org.apache.httpcomponents</groupId>
       <artifactId>httpclient</artifactId>
-      <version>4.5.2</version>
+      <version>4.5.12</version>
```

Comment:
This is not correct, in Fuse we use https://github.com/keycloak/keycloak/blob/master/pom.xml#L1700-L1701

Reply:
@martin-kanis Right, great catch Martin, thanks! The script have their versions mangled, because at the moment there are two duplicate definitions of both the `org.apache.httpcomponents:httpclient` and `org.apachehttpcomponents.httpcore` artifacts in the Keycloak / RH-SSO Fuse adapter license files:

- The first definition came from [[KEYCLOAK-11764] Upgrade to Wildfly 19](https://github.com/keycloak/keycloak/commit/d4eeed306be1bb0a605c5d1154c33a1aabc7fb96#diff-563f018fb700a0ff7c27d000892069ce2e96474417d2d9eb17ec62fa1e4b5382L106) change
- While the second from [[KEYCLOAK-15892] Can not install 7.4.3.CR1 Fuse adapter](https://github.com/keycloak/keycloak/commit/c8d0f2c59cddd3a719a6c03c8d1b267eeddd50c6#diff-563f018fb700a0ff7c27d000892069ce2e96474417d2d9eb17ec62fa1e4b5382R128) change

Since assuming there can be just one version of a specific artifact defined in the license files (if there are two versions of the same, how would be determined which of them should be actually used), tried to consolidate them via 67dfcac commit (intentionally left it separated so you can better double check / verify) -- basically:

1. Dropped the duplicate entries (from both the respective _license.xml_ files and from the *.txt files),
2. And kept only former versions of these artifacts for both the Keycloak and RH-SSO adapters (or more exactly, those versions listed in ```apache.httpcomponents.fuse.version``` and ```apache.httpcomponents.httpcore.fuse.version``` properties)

Please double-check if the state after the 67dfcac commit is the expected one (IOW there should be just one version of those artifacts listed in Fuse adapter files & their versions should match the aforementioned two properties)

Also filed KEYCLOAK-16200 to prevent re-occurrence of duplicate entries in license files, and KEYCLOAK-16202 not to have versions of `httpclient` / `httpcore` in Fuse adapter updated, if it gets updated as part of the Wildfly upgrade (IOW let them get upgraded on other places within the repo, for the purpose of e.g. the testsuite or quarkus reference, but skip the Fuse adapter entries).

Reply:
@iankko Thanks for the update. I think the state of 67dfcac is correct one i.e my understanding is that Fuse licenses should contain entries **only** for httpcomponents used in Fuse (4.5.2 and 4.4.4)  and **don't contain** versions from EAP. @drichtarik is that correct?

Reply:
Previously checked this with @pskopek Per the conclusion listing the same artifact in the same license file with two different versions would be an error. This is already fixed, resolving